### DISCUSSION TITLE
How can I close the information box that appears on the Boot screen?

### DISCUSSION THREAD

**Discussion by yahyacansahan** [mriscoc/Ender3V2S1#1040] [Q&A]
### Did you test with a precompiled firmware?

Yes, and the problem still exists.

### Bug Description

It takes up most of the custom image I added and prevents it from appearing. Is there a way to turn it off?
Although I searched for a long time in the settings file, I could not find a way to turn it off.

### Version of Professional Firmware

2.1.3

### Bed Leveling

None

### Your Slicer

None

### Host Software

None

**Comment by mriscoc** [comment]
The only way is disabling the PROUI_EX.
